openqabot/loader/config.py

```diff
@@ -32,7 +38,7 @@ def load_metadata(
         try:
             settings = data.get("settings")
         except AttributeError:
-            # not valid yaml for bot settings
+            log.error("The YAML file '%s' contains no valid data for bot settings.", p)
```

Comment:
suggestion: `"The YAML file '%s' contains invalid settings."

Reply:
That's not correct, it is possible the yaml is missing some data but it should not contain invalid one as the pipeline for metadata should check for that. And there is a distinction between invalid settings and no settings.